README.md

```diff
@@ -49,17 +49,65 @@ There is no return value from this method.
 
 If language selection fails (no matching languages), the callback will contain an error.
 
+
+#### `.transformByField({source}, [languages], {fields});`
+
+This method applies the transform on a field by field basis as opposed to comparing a whole `i18n`
+block like the `transform` method does.
+
+The `fields` object should adhere to the following schema:
+
+```
+{
+    "required": [
```

Comment:
Quotes: Ok, I was wondering why "required" has quotes and optional doesn't :)

Fields: ok, maybe we can make it explicit that we will ignore anything that is not in the list. What happens if we put a field in both lists?